latte/cs/tags.texy

```diff
@@ -570,7 +570,7 @@ Sandboxovanou obdobou `{include}` je:
 =========
 
 Pojmenované bloky jsou základní stavební jednotkou [dědičnosti a znovupoužitelnost šablon|template-inheritance]. Avšak bloky bez jména slouží
-jako způsob jak aplikovat [filtry|language#filtry] na část šablony. Například takto lze aplikovat filtr [strip|filters#strip], který odstraní zbytečné mezery:
+jako způsob jak aplikovat [filtry|filters] na část šablony. Například takto lze aplikovat filtr [strip|filters#strip], který odstraní zbytečné mezery:
```

Comment:
proč myslíš, že je lepší odkaz na filters než ten původní?

Reply:
Kdyz na to klikam, tak me to hodi sem, https://latte.nette.org/cs/language#toc-filtry, coz je 404.

Reply:
Aha! ono tam ma být syntax#filtry

Reply:
Ja prave nevedel kam presne to ma ukazovat, ale lepsi nez 404. :-) Klidne to cherry pickni dle potreby.